### ISSUE TITLE
Clarification on reported dev numbers on GLUE tasks in Table 4 of the paper

### ISSUE THREAD

**Issue by ngoyal2707** [zihangdai/xlnet#96]
Hi, congrats on great paper and results. 
I was wondering if you can clear couple of questions about the setup for reported numbers in Table 4, Row 2 of the paper. 

1) What was the hyperparam search space you guys searched for finetuning on each task?

2) Are the reported numbers best or average or median across seeds? (or just 1 seed?)

Thanks again for taking time to answer these questions.

**Comment by zihangdai** [comment]
QQP: bsz=128, lr=3e-5, train_steps=14000, warmup_step=1000, eval_all_ckpt=True
MNLI: bsz=128, lr=3e-5, train_steps=10000, warmup_step=1000, eval_all_ckpt=True
SST-2: bsz=32, lr=2e-5, train_steps=10000, warmup_step=1000, eval_all_ckpt=True
CoLA: bsz=128, lr=3e-5, train_steps=3000, warmup_step=500, eval_all_ckpt=True
STS-B: bsz=32, lr=5e-5, train_steps=1200, warmup_step=120, eval_all_ckpt=True
MRPC: bsz=32, lr=5e-5, train_steps=800, warmup_step=200, eval_all_ckpt=True
RTE: bsz=32, lr=3e-5, train_steps=800, warmup_step=200, eval_all_ckpt=True

All results are the median of 5 runs. For smaller datasets, the variance is larger. MNLI/QQP are very stable.

**Comment by ngoyal2707** [comment]
Thanks for such quick answer, it helps a lot.

**Comment by cockroachzl** [comment]
@zihangdai Would be helpful to many persons if the above hyperparameters were added to the readme of this repo! Thanks!
